webapi/_test_data/all_data/webern-data.ttl

```diff
@@ -0,0 +1,204 @@
+@prefix : <http://www.knora.org/data/08AE/webern#> .
+@prefix dc: <http://www.knora.org/ontology/dc#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
+@prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
+@prefix webern: <http://www.knora.org/ontology/08AE/webern#> .
+
+#
+# 
+# 
+# #################################################################
+# #
+# #    Individuals
+# #
+# #################################################################
+# 
+# 
+# http://data.knora.org/webern/testperson1
+
+<http://data.knora.org/webern/testperson1> a webern:Person, owl:NamedIndividual ;
+	webern:comment <http://data.knora.org/webern/testperson1/values/df1260ad43d5> ;
```

Comment:
The value IRI: `http://rdfh.ch/08AE/testperson1/values/df1260ad43d5`

Reply:
Changed it accordingly, but UUIDs are not systematically matching the specs' requirements.